src/content/CloudBoilerplateNet/Services/CachedDeliveryClient.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace CloudBoilerplateNet.Services
 {
-    public class CachedDeliveryClient : IDeliveryClient
+    public class CachedDeliveryClient : IDeliveryClient, IDisposable
```

Comment:
A base class with subclasses that should be disposable must implement IDisposable as follows. You should use this pattern whenever you implement IDisposable on any type that isn't sealed (NotInheritable in Visual Basic).
https://msdn.microsoft.com/en-us/library/system.idisposable(v=vs.110).aspx